src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Lottery {
+    public void getRandomBall() {
```

Comment:
let the Lottery#getRandomBall method returns the Ball, not a void.

The name of the method calls to us that we expect to `get` something, but we won't.